stubs/Authlib/METADATA.toml

```
@@ -1,4 +1,4 @@
 version = "~= 1.6.4"
 upstream_repository = "https://github.com/lepture/authlib"
-requires = ["cryptography"]
+requires = ["cryptography", "httpx", "types-httpx"]
```

```suggestion
requires = ["cryptography", "httpx"]
```

there's no such thing as `types-httpx`, because `httpx` is typed.